docs/pages/ktabslist.vue

```diff
@@ -386,34 +268,25 @@
       </ul>
     </DocsPageSection>
   </DocsPageTemplate>
-
 </template>
 
-
 <script>
-
-  export default {
-    data() {
-      return {
-        tabs: [
-          { id: 'tabLessons', label: 'Lessons' },
-          { id: 'tabLearners', label: 'Learners' },
-          { id: 'tabGroups', label: 'Groups' },
-        ],
-        ex1activeTabId: 'tabLessons',
-        ex2activeTabId: 'tabLessons',
-        ex3activeTabId: 'tabLessons',
-        ex4activeTabId: 'tabLessons',
-        icons: {
-          tabLessons: 'lesson',
-          tabLearners: 'person',
-          tabGroups: 'people',
-        },
-      };
-    },
-  };
-
-</script>
-
-
-<style lang="scss" scoped></style>
+export default {
+  name: 'DocsKTabsList',
+  data() {
+    return {
+      tabs: [
+        { id: 'tabLessons', label: 'Lessons' },
+        { id: 'tabLearners', label: 'Learners' },
+        { id: 'tabGroups', label: 'Groups' },
+      ],
+      ex1activeTabId: 'tabLessons',
+      icons: {
+        tabLessons: 'lesson',
+        tabLearners: 'person',
+        tabGroups: 'people',
+      },
+    };
+  },
+};
```

Comment:
Can we remove all these exported data members (as all of the examples and the code needed to run them has been migrated to individual files)?